README.md

```diff
@@ -33,7 +33,7 @@
   <ul>
```

Comment:
- [ ] Kindly remove the following broken links from your table of contents as they point to subsections that have been removed. This will make your readme file more professional.

![image](https://github.com/Chimoneg27/Personal-Portfolio/assets/81932478/b30370b4-131b-42d5-adb4-5643f8ed23fc)